### ISSUE TITLE
Rogue:You are in shapeshift form (mangos one)

### ISSUE THREAD

**Issue by Mibu86** [CoronaCore/Issues#198]
every time i try to mount with my rogue wile i am in stealth : "you are in shapeshift form"
![wowscrnshot_102714_163127](https://cloud.githubusercontent.com/assets/6356589/4792400/1799a9bc-5de6-11e4-9dc9-51af3914f8e7.jpg)

**Comment by Salja** [comment]
Yes very very old mistske bug xD :-) i will push a workround in few days
